snmp_dell/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: snmp_dell
+
+## Overview
+
+Dell Inc. is a technology company that develops computers and network hardware for high performance networking in both small large businesses. Collect metrics for monitoring and alerting from Dell hardware including:
+
+* Dell PowerEdge
+* Dell iDRAC
+* Dell EMC Isilon
+
+For a complete list of all the metrics collected from Dell devices, see the [Network Device Monitoring documentation][1]
+
+## Setup
+
+To install and configure the SNMP integration, see the [Network Device Monitoring][2] documentation.
+
+## Data Collected
+
+### Metrics
+
+For details of monitored metrics see the [Network Device Monitoring documentation][1].
+
+### Service Checks
+
+Snmp Dell does not include any service checks.
```

Comment:
nit, but can we remove this from saying snmp dell and just say the dell integration does not include any service checks?